doc/api/fs.md

```diff
@@ -3815,10 +3815,10 @@ to be notified of filesystem changes.
 * On Aix systems, this feature depends on [`AHAFS`][], which must be enabled.
 
 If the underlying functionality is not available for some reason, then
-`fs.watch` will not be able to function. For example, watching files or
-directories can be unreliable, and in some cases impossible, on network file
-systems (NFS, SMB, etc), or host file systems when using virtualization software
-such as Vagrant, Docker, etc.
+`fs.watch` will not be able to function (an exception may be thrown).
+For example, watching files or directories can be unreliable, and in some
+cases impossible, on network file systems (NFS, SMB, etc), or host file systems
+when using virtualization software such as Vagrant, Docker, etc.
```

Comment:
```suggestion
when using virtualization software such as Vagrant or Docker.
```

Reply:
Thanks for reviewing! it is fixed now in 9c5e8fa